include/xtt/util/tpm_handles.h

```diff
@@ -0,0 +1,14 @@
+#ifndef XTT_UTIL_TPM_HANDLES_H
+#define XTT_UTIL_TPM_HANDLES_H
+#pragma once
+
+#define key_handle_g 0x81800000
```

Comment:
`define`s aren't visible to wrapper libraries (e.g., `xtt-python`), so we should add accessor functions too.

See this [example](https://github.com/xaptum/xtt/blob/eb1b4c6fc768bf247446724725ec3758bf27af97/include/xtt/messages.h#L31) for the message lengths.  (The example is missing the "xtt" prefix on the names;  that's a mistake.)